Lib/test/test_inspect.py

```diff
@@ -1986,6 +1986,41 @@ def test(a, b:'foo'=10, *args:'bar', spam:'baz', ham=123, **kwargs:int):
                            ('kwargs', ..., int, "var_keyword")),
                           ...))
 
+    def test_signature_without_self(self):
+        def test_args_only(*args):
+            pass
```

Comment:
@serhiy-storchaka @1st1 
Codecov complains that they want to add unit test codes for these functions which are just only for test.
I think that add unit tests code for test code is not reasonable.
What do you think?